---
assembly_status: <em style="color:orangered">Draft</em>
common_name: ''
data_arima-1_bases: 280.06 Gbp
data_arima-1_bytes: 145.5 GiB
data_arima-1_coverage: N/A
data_arima-1_links: s3://genomeark/species/Lestoros_inca/mLesInc1/genomic_data/arima/<br>
data_arima-1_s3gui: https://42basepairs.com/browse/s3/genomeark/species/Lestoros_inca/mLesInc1/genomic_data/arima/
data_arima-1_s3url: https://genomeark.s3.amazonaws.com/index.html?prefix=species/Lestoros_inca/mLesInc1/genomic_data/arima/
data_arima-1_scale: 1.7922
data_arima_bases: 280.06 Gbp
data_arima_bytes: 145.5 GiB
data_arima_coverage: N/A
data_pacbiohifi_bam-1_bases: 222.62 Gbp
data_pacbiohifi_bam-1_bytes: 716.4 GiB
data_pacbiohifi_bam-1_coverage: N/A
data_pacbiohifi_bam-1_links: s3://genomeark/species/Lestoros_inca/mLesInc1/genomic_data/pacbio_hifi/<br>
data_pacbiohifi_bam-1_s3gui: https://42basepairs.com/browse/s3/genomeark/species/Lestoros_inca/mLesInc1/genomic_data/pacbio_hifi/
data_pacbiohifi_bam-1_s3url: https://genomeark.s3.amazonaws.com/index.html?prefix=species/Lestoros_inca/mLesInc1/genomic_data/pacbio_hifi/
data_pacbiohifi_bam-1_scale: 0.2894
data_pacbiohifi_bam_bases: 222.62 Gbp
data_pacbiohifi_bam_bytes: 716.4 GiB
data_pacbiohifi_bam_coverage: N/A
data_pacbiohifi_fqgz-1_bases: 217.93 Gbp
data_pacbiohifi_fqgz-1_bytes: 97.6 GiB
data_pacbiohifi_fqgz-1_coverage: N/A
data_pacbiohifi_fqgz-1_links: s3://genomeark/species/Lestoros_inca/mLesInc1/genomic_data/pacbio_hifi/<br>
data_pacbiohifi_fqgz-1_s3gui: https://42basepairs.com/browse/s3/genomeark/species/Lestoros_inca/mLesInc1/genomic_data/pacbio_hifi/
data_pacbiohifi_fqgz-1_s3url: https://genomeark.s3.amazonaws.com/index.html?prefix=species/Lestoros_inca/mLesInc1/genomic_data/pacbio_hifi/
data_pacbiohifi_fqgz-1_scale: 2.0805
data_pacbiohifi_fqgz_bases: 217.93 Gbp
data_pacbiohifi_fqgz_bytes: 97.6 GiB
data_pacbiohifi_fqgz_coverage: N/A
data_status: '<em style="color:forestgreen">PacBio HiFi</em> ::: <em style="color:forestgreen">Arima</em>'
data_use_source: from-default
data_use_text: Samples and data come from a variety of sources. To support fair and
  productive use of this data, please abide by the <a href="https://genome10k.soe.ucsc.edu/data-use-policies/">Data
  Use Policy</a> and contact Erich D. Jarvis, ejarvis@rockefeller.edu, with any questions.
genome_size: 0
genome_size_display: ''
genome_size_method: ''
hpa1date: 2025-07-23
hpa1filesize: 1.1 GiB
hpa1length: '4246459560'
hpa1n50ctg: 11940304
hpa1n50scf: 851326621
hpa1seq: https://s3.amazonaws.com/genomeark/species/Lestoros_inca/mLesInc1/assembly_vgp_HiC_2.0/mLesInc1.HiC.hap1.20250723.fasta.gz
hpa1sizes: |
  <table class="sequence-sizes-table">
  <thead>
  <tr>
  <th></th>
  <th colspan=2 align=center>Contigs</th>
  <th colspan=2 align=center>Scaffolds</th>
  </tr>
  <tr>
  <th>NG</th>
  <th>LG</th>
  <th>Len</th>
  <th>LG</th>
  <th>Len</th>
  </tr>
  </thead>
  <tbody>
  <tr><td> 10 </td><td> 11 </td><td> 30.67 Mbp </td><td> 1 </td><td> 0.96 Gbp </td></tr><tr><td> 20 </td><td> 27 </td><td> 23.31 Mbp </td><td> 1 </td><td> 0.96 Gbp </td></tr><tr><td> 30 </td><td> 48 </td><td> 17.74 Mbp </td><td> 2 </td><td> 0.87 Gbp </td></tr><tr><td> 40 </td><td> 75 </td><td> 14.52 Mbp </td><td> 2 </td><td> 0.87 Gbp </td></tr><tr style="background-color:#cccccc;"><td> 50 </td><td> 107 </td><td style="background-color:#88ff88;"> 11.94 Mbp </td><td> 3 </td><td style="background-color:#88ff88;"> 0.85 Gbp </td></tr><tr><td> 60 </td><td> 148 </td><td> 8.79 Mbp </td><td> 3 </td><td> 0.85 Gbp </td></tr><tr><td> 70 </td><td> 206 </td><td> 5.99 Mbp </td><td> 4 </td><td> 0.64 Gbp </td></tr><tr><td> 80 </td><td> 290 </td><td> 4.21 Mbp </td><td> 5 </td><td> 380.60 Mbp </td></tr><tr><td> 90 </td><td> 435 </td><td> 1.99 Mbp </td><td> 6 </td><td> 372.75 Mbp </td></tr><tr><td> 100 </td><td> 1268 </td><td> 8.12 Kbp </td><td> 224 </td><td> 8.12 Kbp </td></tr></tbody>
  <tfoot>
  <tr><th> 1.000x </th><th> 1268 </th><th> 4.25 Gbp </th><th> 224 </th><th> 4.25 Gbp </th></tr>
  </tfoot>
  </table>
hpa1version: assembly_vgp_HiC_2.0
hpb1date: 2025-07-23
hpb1filesize: 1.1 GiB
hpb1length: '4095655309'
hpb1n50ctg: 11196828
hpb1n50scf: 872700620
hpb1seq: https://s3.amazonaws.com/genomeark/species/Lestoros_inca/mLesInc1/assembly_vgp_HiC_2.0/mLesInc1.HiC.hap2.20250723.fasta.gz
hpb1sizes: |
  <table class="sequence-sizes-table">
  <thead>
  <tr>
  <th></th>
  <th colspan=2 align=center>Contigs</th>
  <th colspan=2 align=center>Scaffolds</th>
  </tr>
  <tr>
  <th>NG</th>
  <th>LG</th>
  <th>Len</th>
  <th>LG</th>
  <th>Len</th>
  </tr>
  </thead>
  <tbody>
  <tr><td> 10 </td><td> 11 </td><td> 32.94 Mbp </td><td> 1 </td><td> 1.80 Gbp </td></tr><tr><td> 20 </td><td> 24 </td><td> 27.29 Mbp </td><td> 1 </td><td> 1.80 Gbp </td></tr><tr><td> 30 </td><td> 42 </td><td> 18.64 Mbp </td><td> 1 </td><td> 1.80 Gbp </td></tr><tr><td> 40 </td><td> 66 </td><td> 14.93 Mbp </td><td> 1 </td><td> 1.80 Gbp </td></tr><tr style="background-color:#cccccc;"><td> 50 </td><td> 98 </td><td style="background-color:#88ff88;"> 11.20 Mbp </td><td> 2 </td><td style="background-color:#88ff88;"> 0.87 Gbp </td></tr><tr><td> 60 </td><td> 141 </td><td> 8.33 Mbp </td><td> 2 </td><td> 0.87 Gbp </td></tr><tr><td> 70 </td><td> 197 </td><td> 6.44 Mbp </td><td> 3 </td><td> 0.63 Gbp </td></tr><tr><td> 80 </td><td> 278 </td><td> 4.03 Mbp </td><td> 3 </td><td> 0.63 Gbp </td></tr><tr><td> 90 </td><td> 415 </td><td> 2.07 Mbp </td><td> 5 </td><td> 371.43 Mbp </td></tr><tr><td> 100 </td><td> 1104 </td><td> 13.43 Kbp </td><td> 116 </td><td> 13.43 Kbp </td></tr></tbody>
  <tfoot>
  <tr><th> 1.000x </th><th> 1104 </th><th> 4.10 Gbp </th><th> 116 </th><th> 4.10 Gbp </th></tr>
  </tfoot>
  </table>
hpb1version: assembly_vgp_HiC_2.0
last_raw_data: 1749578605
last_updated: 1753294954
mds:
- data: species:&nbsp;Lestoros&nbsp;inca<br>specimem:&nbsp;mLesInc1<br>projects:<br>&nbsp;&nbsp;-&nbsp;vgp<br>assembled_by_group:&nbsp;Rockefeller<br>data_location:&nbsp;S3<br>release_to:&nbsp;S3<br>combine_for_curation:&nbsp;true<br>hap1:&nbsp;s3://genomeark/species/Lestoros_inca/mLesInc1/assembly_vgp_HiC_2.0/mLesInc1.HiC.hap1.20250723.fasta.gz<br>hap2:&nbsp;s3://genomeark/species/Lestoros_inca/mLesInc1/assembly_vgp_HiC_2.0/mLesInc1.HiC.hap2.20250723.fasta.gz<br>pretext_hap1:&nbsp;s3://genomeark/species/Lestoros_inca/mLesInc1/assembly_vgp_HiC_2.0/evaluation/hap1/pretext/s2/mLesInc1_hap1__s2_heatmap.pretext<br>pretext_hap2:&nbsp;s3://genomeark/species/Lestoros_inca/mLesInc1/assembly_vgp_HiC_2.0/evaluation/hap2/pretext/s2/mLesInc1_hap2__s2_heatmap.pretext<br>kmer_spectra_img:&nbsp;s3://genomeark/species/Lestoros_inca/mLesInc1/assembly_vgp_HiC_2.0/evaluation/merqury/mLesInc1_png/<br>pacbio_read_dir:&nbsp;s3://genomeark/species/Lestoros_inca/mLesInc1/genomic_data/pacbio_hifi/<br>pacbio_read_type:&nbsp;hifi<br>hic_read_dir:&nbsp;s3://genomeark/species/Lestoros_inca/mLesInc1/genomic_data/arima/<br>pipeline:<br>&nbsp;&nbsp;-&nbsp;hifiasm&nbsp;(0.25.0+galaxy0)<br>&nbsp;&nbsp;-&nbsp;yahs&nbsp;(1.2a.2+galaxy2)<br>&nbsp;&nbsp;-&nbsp;VGP0&nbsp;mitoHiFi&nbsp;workflow&nbsp;(v0.2.1)<br>&nbsp;&nbsp;-&nbsp;VGP1&nbsp;genomic&nbsp;profiling&nbsp;workflow&nbsp;(v0.3)<br>&nbsp;&nbsp;-&nbsp;VGP4&nbsp;HiFi&nbsp;HiC&nbsp;assembly&nbsp;workflow&nbsp;(v0.3.8)<br>&nbsp;&nbsp;-&nbsp;VGP8&nbsp;HiC&nbsp;scaffolding&nbsp;workflow&nbsp;(v2.0)<br>&nbsp;&nbsp;-&nbsp;VGP9&nbsp;decontamination&nbsp;workflow&nbsp;(v0.7)&nbsp;with&nbsp;database&nbsp;PlusPF&nbsp;2022<br>&nbsp;&nbsp;-&nbsp;Pre-curation&nbsp;worklfow&nbsp;(v1.0beta6)<br>notes:&nbsp;This&nbsp;was&nbsp;a&nbsp;Hifiasm-HiC&nbsp;assembly&nbsp;of&nbsp;mLesInc1&nbsp;resulting&nbsp;in&nbsp;two&nbsp;haplotype&nbsp;assemblies.&nbsp;HiC&nbsp;data&nbsp;were&nbsp;generated&nbsp;using&nbsp;the&nbsp;Arima&nbsp;library&nbsp;prep&nbsp;kit,&nbsp;and&nbsp;require&nbsp;trimming&nbsp;the&nbsp;first&nbsp;5&nbsp;bp&nbsp;off&nbsp;from&nbsp;the&nbsp;5'&nbsp;end&nbsp;of&nbsp;both&nbsp;read&nbsp;files.&nbsp;The&nbsp;restriction&nbsp;enzymes&nbsp;used&nbsp;were&nbsp;the&nbsp;Arima&nbsp;v2&nbsp;kit,&nbsp;so&nbsp;the&nbsp;cutting&nbsp;sites&nbsp;are&nbsp;GATC,&nbsp;GANTC,&nbsp;CTNAG,&nbsp;TTAA.<br>
  ident: md4
  title: assembly_vgp_HiC_2.0/mLesInc1.yml
mito1date: 2025-07-23
mito1filesize: 5.3 KiB
mito1length: '16699'
mito1n50ctg: 16699
mito1n50scf: 16699
mito1seq: https://s3.amazonaws.com/genomeark/species/Lestoros_inca/mLesInc1/assembly_MT_rockefeller/mLesInc1.MT.20250723.fasta.gz
mito1sizes: |
  <table class="sequence-sizes-table">
  <thead>
  <tr>
  <th></th>
  <th colspan=2 align=center>Contigs</th>
  <th colspan=2 align=center>Scaffolds</th>
  </tr>
  <tr>
  <th>NG</th>
  <th>LG</th>
  <th>Len</th>
  <th>LG</th>
  <th>Len</th>
  </tr>
  </thead>
  <tbody>
  <tr><td> 10 </td><td> 1 </td><td> 16.70 Kbp </td><td> 1 </td><td> 16.70 Kbp </td></tr><tr><td> 20 </td><td> 1 </td><td> 16.70 Kbp </td><td> 1 </td><td> 16.70 Kbp </td></tr><tr><td> 30 </td><td> 1 </td><td> 16.70 Kbp </td><td> 1 </td><td> 16.70 Kbp </td></tr><tr><td> 40 </td><td> 1 </td><td> 16.70 Kbp </td><td> 1 </td><td> 16.70 Kbp </td></tr><tr style="background-color:#cccccc;"><td> 50 </td><td> 1 </td><td style="background-color:#ff8888;"> 16.70 Kbp </td><td> 1 </td><td style="background-color:#ff8888;"> 16.70 Kbp </td></tr><tr><td> 60 </td><td> 1 </td><td> 16.70 Kbp </td><td> 1 </td><td> 16.70 Kbp </td></tr><tr><td> 70 </td><td> 1 </td><td> 16.70 Kbp </td><td> 1 </td><td> 16.70 Kbp </td></tr><tr><td> 80 </td><td> 1 </td><td> 16.70 Kbp </td><td> 1 </td><td> 16.70 Kbp </td></tr><tr><td> 90 </td><td> 1 </td><td> 16.70 Kbp </td><td> 1 </td><td> 16.70 Kbp </td></tr><tr><td> 100 </td><td> 1 </td><td> 16.70 Kbp </td><td> 1 </td><td> 16.70 Kbp </td></tr></tbody>
  <tfoot>
  <tr><th> 1.000x </th><th> 1 </th><th> 16.70 Kbp </th><th> 1 </th><th> 16.70 Kbp </th></tr>
  </tfoot>
  </table>
mito1version: assembly_MT_rockefeller
name: Lestoros inca
name_: Lestoros_inca
project: ~
short_name: mLesInc
taxon_id: ''
---
