---
assembly_status: <em style="color:forestgreen">Curated</em>
common_name: ''
data_arima-2_bases: 125.43 Gbp
data_arima-2_bytes: 61.5 GiB
data_arima-2_coverage: N/A
data_arima-2_links: s3://genomeark/species/Morus_bassanus/bMorBas2/genomic_data/arima/<br>
data_arima-2_s3gui: https://42basepairs.com/browse/s3/genomeark/species/Morus_bassanus/bMorBas2/genomic_data/arima/
data_arima-2_s3url: https://genomeark.s3.amazonaws.com/index.html?prefix=species/Morus_bassanus/bMorBas2/genomic_data/arima/
data_arima-2_scale: 1.9002
data_arima_bases: 125.43 Gbp
data_arima_bytes: 61.5 GiB
data_arima_coverage: N/A
data_pacbiohifi_bam-2_bases: 158.55 Gbp
data_pacbiohifi_bam-2_bytes: 542.1 GiB
data_pacbiohifi_bam-2_coverage: N/A
data_pacbiohifi_bam-2_links: s3://genomeark/species/Morus_bassanus/bMorBas2/genomic_data/pacbio_hifi/<br>
data_pacbiohifi_bam-2_s3gui: https://42basepairs.com/browse/s3/genomeark/species/Morus_bassanus/bMorBas2/genomic_data/pacbio_hifi/
data_pacbiohifi_bam-2_s3url: https://genomeark.s3.amazonaws.com/index.html?prefix=species/Morus_bassanus/bMorBas2/genomic_data/pacbio_hifi/
data_pacbiohifi_bam-2_scale: 0.2724
data_pacbiohifi_bam_bases: 158.55 Gbp
data_pacbiohifi_bam_bytes: 542.1 GiB
data_pacbiohifi_bam_coverage: N/A
data_pacbiohifi_clr-2_bases: 1172.36 Gbp
data_pacbiohifi_clr-2_bytes: 2.0 TiB
data_pacbiohifi_clr-2_coverage: N/A
data_pacbiohifi_clr-2_links: s3://genomeark/species/Morus_bassanus/bMorBas2/genomic_data/pacbio_hifi/<br>
data_pacbiohifi_clr-2_s3gui: https://42basepairs.com/browse/s3/genomeark/species/Morus_bassanus/bMorBas2/genomic_data/pacbio_hifi/
data_pacbiohifi_clr-2_s3url: https://genomeark.s3.amazonaws.com/index.html?prefix=species/Morus_bassanus/bMorBas2/genomic_data/pacbio_hifi/
data_pacbiohifi_clr-2_scale: 0.5347
data_pacbiohifi_clr_bases: 1172.36 Gbp
data_pacbiohifi_clr_bytes: 2.0 TiB
data_pacbiohifi_clr_coverage: N/A
data_pacbiohifi_fqgz-2_bases: 49.62 Gbp
data_pacbiohifi_fqgz-2_bytes: 37.4 GiB
data_pacbiohifi_fqgz-2_coverage: N/A
data_pacbiohifi_fqgz-2_links: s3://genomeark/species/Morus_bassanus/bMorBas2/genomic_data/pacbio_hifi/<br>
data_pacbiohifi_fqgz-2_s3gui: https://42basepairs.com/browse/s3/genomeark/species/Morus_bassanus/bMorBas2/genomic_data/pacbio_hifi/
data_pacbiohifi_fqgz-2_s3url: https://genomeark.s3.amazonaws.com/index.html?prefix=species/Morus_bassanus/bMorBas2/genomic_data/pacbio_hifi/
data_pacbiohifi_fqgz-2_scale: 1.2354
data_pacbiohifi_fqgz_bases: 49.62 Gbp
data_pacbiohifi_fqgz_bytes: 37.4 GiB
data_pacbiohifi_fqgz_coverage: N/A
data_status: '<em style="color:forestgreen">PacBio HiFi</em> ::: <em style="color:forestgreen">Arima</em>'
data_use_source: from-default
data_use_text: Samples and data come from a variety of sources. To support fair and
  productive use of this data, please abide by the <a href="https://genome10k.soe.ucsc.edu/data-use-policies/">Data
  Use Policy</a> and contact Erich D. Jarvis, ejarvis@rockefeller.edu, with any questions.
genome_size: 0
genome_size_display: ''
genome_size_method: ''
hpa2date: 2023-06-27
hpa2filesize: 392.0 MiB
hpa2length: '1381630140'
hpa2n50ctg: 9126423
hpa2n50scf: 67716506
hpa2seq: https://s3.amazonaws.com/genomeark/species/Morus_bassanus/bMorBas2/assembly_curated/bMorBas2.hap1.cur.20230627.fasta.gz
hpa2sizes: |
  <table class="sequence-sizes-table">
  <thead>
  <tr>
  <th></th>
  <th colspan=2 align=center>Contigs</th>
  <th colspan=2 align=center>Scaffolds</th>
  </tr>
  <tr>
  <th>NG</th>
  <th>LG</th>
  <th>Len</th>
  <th>LG</th>
  <th>Len</th>
  </tr>
  </thead>
  <tbody>
  <tr><td> 10 </td><td> 5 </td><td> 27.27 Mbp </td><td> 1 </td><td> 170.32 Mbp </td></tr><tr><td> 20 </td><td> 11 </td><td> 18.70 Mbp </td><td> 2 </td><td> 135.26 Mbp </td></tr><tr><td> 30 </td><td> 19 </td><td> 16.88 Mbp </td><td> 3 </td><td> 116.62 Mbp </td></tr><tr><td> 40 </td><td> 29 </td><td> 12.21 Mbp </td><td> 5 </td><td> 85.79 Mbp </td></tr><tr style="background-color:#cccccc;"><td> 50 </td><td> 42 </td><td style="background-color:#88ff88;"> 9.13 Mbp </td><td> 7 </td><td style="background-color:#88ff88;"> 67.72 Mbp </td></tr><tr><td> 60 </td><td> 61 </td><td> 6.12 Mbp </td><td> 9 </td><td> 45.46 Mbp </td></tr><tr><td> 70 </td><td> 88 </td><td> 4.29 Mbp </td><td> 13 </td><td> 30.86 Mbp </td></tr><tr><td> 80 </td><td> 128 </td><td> 2.70 Mbp </td><td> 19 </td><td> 14.15 Mbp </td></tr><tr><td> 90 </td><td> 207 </td><td> 1.04 Mbp </td><td> 37 </td><td> 2.69 Mbp </td></tr><tr><td> 100 </td><td> 1092 </td><td> 10.74 Kbp </td><td> 829 </td><td> 10.74 Kbp </td></tr></tbody>
  <tfoot>
  <tr><th> 1.000x </th><th> 1092 </th><th> 1.38 Gbp </th><th> 829 </th><th> 1.38 Gbp </th></tr>
  </tfoot>
  </table>
hpa2version: assembly_curated
hpb2date: 2023-06-27
hpb2filesize: 373.8 MiB
hpb2length: '1294787636'
hpb2n50ctg: 10178591
hpb2n50scf: 85624757
hpb2seq: https://s3.amazonaws.com/genomeark/species/Morus_bassanus/bMorBas2/assembly_curated/bMorBas2.hap2.cur.20230627.fasta.gz
hpb2sizes: |
  <table class="sequence-sizes-table">
  <thead>
  <tr>
  <th></th>
  <th colspan=2 align=center>Contigs</th>
  <th colspan=2 align=center>Scaffolds</th>
  </tr>
  <tr>
  <th>NG</th>
  <th>LG</th>
  <th>Len</th>
  <th>LG</th>
  <th>Len</th>
  </tr>
  </thead>
  <tbody>
  <tr><td> 10 </td><td> 4 </td><td> 27.06 Mbp </td><td> 1 </td><td> 170.18 Mbp </td></tr><tr><td> 20 </td><td> 10 </td><td> 19.00 Mbp </td><td> 2 </td><td> 135.25 Mbp </td></tr><tr><td> 30 </td><td> 19 </td><td> 14.02 Mbp </td><td> 3 </td><td> 117.81 Mbp </td></tr><tr><td> 40 </td><td> 28 </td><td> 12.17 Mbp </td><td> 5 </td><td> 85.67 Mbp </td></tr><tr style="background-color:#cccccc;"><td> 50 </td><td> 40 </td><td style="background-color:#88ff88;"> 10.18 Mbp </td><td> 6 </td><td style="background-color:#88ff88;"> 85.62 Mbp </td></tr><tr><td> 60 </td><td> 55 </td><td> 7.60 Mbp </td><td> 8 </td><td> 46.53 Mbp </td></tr><tr><td> 70 </td><td> 75 </td><td> 4.87 Mbp </td><td> 11 </td><td> 37.27 Mbp </td></tr><tr><td> 80 </td><td> 106 </td><td> 3.48 Mbp </td><td> 15 </td><td> 25.35 Mbp </td></tr><tr><td> 90 </td><td> 165 </td><td> 1.44 Mbp </td><td> 24 </td><td> 9.16 Mbp </td></tr><tr><td> 100 </td><td> 668 </td><td> 9.93 Kbp </td><td> 420 </td><td> 9.93 Kbp </td></tr></tbody>
  <tfoot>
  <tr><th> 1.000x </th><th> 668 </th><th> 1.29 Gbp </th><th> 420 </th><th> 1.29 Gbp </th></tr>
  </tfoot>
  </table>
hpb2version: assembly_curated
last_raw_data: 1718386381
last_updated: 1718386381
mds:
- data: species:&nbsp;Morus&nbsp;bassanus<br>specimen:&nbsp;bMorBas2<br>projects:&nbsp;<br>&nbsp;&nbsp;-&nbsp;vgp<br>data_location:&nbsp;S3<br>release_to:&nbsp;S3<br>haplotype_to_curate:&nbsp;hap1<br>hap1:&nbsp;s3://genomeark/species/Morus_bassanus/bMorBas2/assembly_vgp_HiC_2.0/bMorBas2.HiC.hap1.20230417.fasta.gz<br>hap2:&nbsp;s3://genomeark/species/Morus_bassanus/bMorBas2/assembly_vgp_HiC_2.0/bMorBas2.HiC.hap2.20230417.fasta.gz<br>pretext_hap1:&nbsp;s3://genomeark/species/Morus_bassanus/bMorBas2/assembly_vgp_HiC_2.0/evaluation/hap1/pretext/bMorBas2_hap1__s2_heatmap.pretext<br>pretext_hap2:&nbsp;s3://genomeark/species/Morus_bassanus/bMorBas2/assembly_vgp_HiC_2.0/evaluation/hap2/pretext/bMorBas2_hap2__s2_heatmap.pretext<br>kmer_spectra_img:&nbsp;s3://genomeark/species/Morus_bassanus/bMorBas2/assembly_vgp_HiC_2.0/evaluation/merqury/bMorBas2_png/<br>mito:&nbsp;s3://genomeark/species/Morus_bassanus/bMorBas2/assembly_MT_rockefeller/bMorBas2.MT.20230411.fasta.gz<br>pacbio_read_dir:&nbsp;s3://genomeark/species/Morus_bassanus/bMorBas2/genomic_data/pacbio_hifi/<br>pacbio_read_type:&nbsp;hifi<br>hic_read_dir:&nbsp;s3://genomeark/species/Morus_bassanus/bMorBas2/genomic_data/arima/<br>pipeline:<br>&nbsp;&nbsp;-&nbsp;hifiasm&nbsp;(0.19.3+galaxy0)<br>&nbsp;&nbsp;-&nbsp;yahs&nbsp;(1.2a.2+galaxy0)<br>assembled_by_group:&nbsp;Rockefeller<br>notes:&nbsp;This&nbsp;was&nbsp;a&nbsp;hifiasm-HiC&nbsp;assembly&nbsp;of&nbsp;bMorBas2&nbsp;(VGL&nbsp;ID&nbsp;is&nbsp;VGL-bMorBas1),&nbsp;resulting&nbsp;in&nbsp;two&nbsp;complete&nbsp;haplotypes.&nbsp;This&nbsp;individual&nbsp;did&nbsp;not&nbsp;have&nbsp;bionano&nbsp;data.&nbsp;HiC&nbsp;scaffolding&nbsp;was&nbsp;performed&nbsp;with&nbsp;yahs.&nbsp;K-mer&nbsp;spectra&nbsp;indicates&nbsp;a&nbsp;homogametic&nbsp;specimen.&nbsp;The&nbsp;HiC&nbsp;prep&nbsp;was&nbsp;Arima&nbsp;kit&nbsp;2.&nbsp;I&nbsp;am&nbsp;submitting&nbsp;both&nbsp;hap1&nbsp;&&nbsp;hap2&nbsp;for&nbsp;dual&nbsp;curation.&nbsp;This&nbsp;is&nbsp;the&nbsp;curation&nbsp;ticket&nbsp;for&nbsp;hap1.&nbsp;
  ident: md9
  title: assembly_vgp_HiC_2.0/bMorBas2_hap1.yml
- data: species:&nbsp;Morus&nbsp;bassanus<br>specimen:&nbsp;bMorBas2<br>projects:&nbsp;<br>&nbsp;&nbsp;-&nbsp;vgp<br>data_location:&nbsp;S3<br>release_to:&nbsp;S3<br>haplotype_to_curate:&nbsp;hap1<br>hap1:&nbsp;s3://genomeark/species/Morus_bassanus/bMorBas2/assembly_vgp_HiC_2.0/bMorBas2.HiC.hap1.20230417.fasta.gz<br>hap2:&nbsp;s3://genomeark/species/Morus_bassanus/bMorBas2/assembly_vgp_HiC_2.0/bMorBas2.HiC.hap2.20230417.fasta.gz<br>pretext_hap1:&nbsp;s3://genomeark/species/Morus_bassanus/bMorBas2/assembly_vgp_HiC_2.0/evaluation/hap1/pretext/bMorBas2_hap1__s2_heatmap.pretext<br>pretext_hap2:&nbsp;s3://genomeark/species/Morus_bassanus/bMorBas2/assembly_vgp_HiC_2.0/evaluation/hap2/pretext/bMorBas2_hap2__s2_heatmap.pretext<br>kmer_spectra_img:&nbsp;s3://genomeark/species/Morus_bassanus/bMorBas2/assembly_vgp_HiC_2.0/evaluation/merqury/bMorBas2_png/<br>mito:&nbsp;s3://genomeark/species/Morus_bassanus/bMorBas2/assembly_MT_rockefeller/bMorBas2.MT.20230411.fasta.gz<br>pacbio_read_dir:&nbsp;s3://genomeark/species/Morus_bassanus/bMorBas2/genomic_data/pacbio_hifi/<br>pacbio_read_type:&nbsp;hifi<br>hic_read_dir:&nbsp;s3://genomeark/species/Morus_bassanus/bMorBas2/genomic_data/arima/<br>pipeline:<br>&nbsp;&nbsp;-&nbsp;hifiasm&nbsp;(0.19.3+galaxy0)<br>&nbsp;&nbsp;-&nbsp;yahs&nbsp;(1.2a.2+galaxy0)<br>assembled_by_group:&nbsp;Rockefeller<br>notes:&nbsp;This&nbsp;was&nbsp;a&nbsp;hifiasm-HiC&nbsp;assembly&nbsp;of&nbsp;bMorBas2&nbsp;(VGL&nbsp;ID&nbsp;is&nbsp;VGL-bMorBas1),&nbsp;resulting&nbsp;in&nbsp;two&nbsp;complete&nbsp;haplotypes.&nbsp;This&nbsp;individual&nbsp;did&nbsp;not&nbsp;have&nbsp;bionano&nbsp;data.&nbsp;HiC&nbsp;scaffolding&nbsp;was&nbsp;performed&nbsp;with&nbsp;yahs.&nbsp;K-mer&nbsp;spectra&nbsp;indicates&nbsp;a&nbsp;homogametic&nbsp;specimen.&nbsp;The&nbsp;HiC&nbsp;prep&nbsp;was&nbsp;Arima&nbsp;kit&nbsp;2.&nbsp;I&nbsp;am&nbsp;submitting&nbsp;both&nbsp;hap1&nbsp;&&nbsp;hap2&nbsp;for&nbsp;dual&nbsp;curation.&nbsp;This&nbsp;is&nbsp;the&nbsp;curation&nbsp;ticket&nbsp;for&nbsp;hap1.&nbsp;
  ident: md10
  title: assembly_vgp_HiC_2.0/bMorBas2_hap2.yml
mito2date: 2023-04-11
mito2filesize: 4.9 KiB
mito2length: '16690'
mito2n50ctg: 16690
mito2n50scf: 16690
mito2seq: https://s3.amazonaws.com/genomeark/species/Morus_bassanus/bMorBas2/assembly_MT_rockefeller/bMorBas2.MT.20230411.fasta.gz
mito2sizes: |
  <table class="sequence-sizes-table">
  <thead>
  <tr>
  <th></th>
  <th colspan=2 align=center>Contigs</th>
  <th colspan=2 align=center>Scaffolds</th>
  </tr>
  <tr>
  <th>NG</th>
  <th>LG</th>
  <th>Len</th>
  <th>LG</th>
  <th>Len</th>
  </tr>
  </thead>
  <tbody>
  <tr><td> 10 </td><td> 1 </td><td> 16.69 Kbp </td><td> 1 </td><td> 16.69 Kbp </td></tr><tr><td> 20 </td><td> 1 </td><td> 16.69 Kbp </td><td> 1 </td><td> 16.69 Kbp </td></tr><tr><td> 30 </td><td> 1 </td><td> 16.69 Kbp </td><td> 1 </td><td> 16.69 Kbp </td></tr><tr><td> 40 </td><td> 1 </td><td> 16.69 Kbp </td><td> 1 </td><td> 16.69 Kbp </td></tr><tr style="background-color:#cccccc;"><td> 50 </td><td> 1 </td><td style="background-color:#ff8888;"> 16.69 Kbp </td><td> 1 </td><td style="background-color:#ff8888;"> 16.69 Kbp </td></tr><tr><td> 60 </td><td> 1 </td><td> 16.69 Kbp </td><td> 1 </td><td> 16.69 Kbp </td></tr><tr><td> 70 </td><td> 1 </td><td> 16.69 Kbp </td><td> 1 </td><td> 16.69 Kbp </td></tr><tr><td> 80 </td><td> 1 </td><td> 16.69 Kbp </td><td> 1 </td><td> 16.69 Kbp </td></tr><tr><td> 90 </td><td> 1 </td><td> 16.69 Kbp </td><td> 1 </td><td> 16.69 Kbp </td></tr><tr><td> 100 </td><td> 1 </td><td> 16.69 Kbp </td><td> 1 </td><td> 16.69 Kbp </td></tr></tbody>
  <tfoot>
  <tr><th> 1.000x </th><th> 1 </th><th> 16.69 Kbp </th><th> 1 </th><th> 16.69 Kbp </th></tr>
  </tfoot>
  </table>
mito2version: assembly_MT_rockefeller
name: Morus bassanus
name_: Morus_bassanus
project: ~
short_name: bMorBas
taxon_id: ''
---
